### PR TITLE
Add Go 1.17 to all OSes toolsets

### PR DESCRIPTION
**Pull request by yuk7**
# Description
Added Go 1.17.* version to all OSes toolsets

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/toolsets/toolset-1604.json [hunk 1]
```diff
@@ -43,7 +43,9 @@
             "versions": [
                 "1.13.*",
                 "1.14.*",
```
Comment (catthehacker) [inline images/linux/toolsets/toolset-1604.json:45]:
```suggestion
```
Remove them, only 3 latest minor versions should be included

File: images/linux/toolsets/toolset-1804.json [hunk 2]
```diff
@@ -43,7 +43,9 @@
             "versions": [
                 "1.13.*",
                 "1.14.*",
```
Comment (catthehacker) [inline images/linux/toolsets/toolset-1804.json:45]:
```suggestion
```
Same as above

File: images/linux/toolsets/toolset-2004.json [hunk 3]
```diff
@@ -44,7 +44,8 @@
             "versions": [
                 "1.14.*",
```
Comment (catthehacker) [inline images/linux/toolsets/toolset-2004.json:45]:
```suggestion
```
Same as above

File: images/macos/toolsets/toolset-10.14.json [hunk 4]
```diff
@@ -315,7 +315,8 @@
                 "1.13.*",
                 "1.14.*",
```
Comment (catthehacker) [inline images/macos/toolsets/toolset-10.14.json:316]:
```suggestion
```
Same as above

File: images/macos/toolsets/toolset-10.15.json [hunk 5]
```diff
@@ -268,7 +268,8 @@
                 "1.13.*",
                 "1.14.*",
```
Comment (catthehacker) [inline images/macos/toolsets/toolset-10.15.json:269]:
```suggestion
```
Same as above

File: images/win/toolsets/toolset-2016.json [hunk 6]
```diff
@@ -72,7 +72,8 @@
                 "1.13.*",
                 "1.14.*",
```
Comment (catthehacker) [inline images/win/toolsets/toolset-2016.json:73]:
```suggestion
```
Same as above

File: images/win/toolsets/toolset-2019.json [hunk 7]
```diff
@@ -72,7 +72,8 @@
                 "1.13.*",
                 "1.14.*",
```
Comment (catthehacker) [inline images/win/toolsets/toolset-2019.json:73]:
```suggestion
```
Same as above

**Review by catthehacker — CHANGES_REQUESTED** [review]

**Comment by yuk7** [comment]
Resolved, Thank you.

**Comment by yuk7** [comment]
Think you for reviewing, I pushed new commit.
Could you check it?

**Comment by catthehacker** [comment]
@maxim-lobanov @miketimofeev

File: images/win/toolsets/toolset-2019.json (hunk 7 above)
Comment (catthehacker) [reply images/win/toolsets/toolset-2019.json:73]: Missed out on `windows-2019`

File: images/linux/toolsets/toolset-1604.json [hunk 8]
```diff
@@ -41,9 +41,9 @@
             "arch": "x64",
             "platform" : "linux",
             "versions": [
-                "1.13.*",
-                "1.14.*",
```
Comment (MaksimZhukov) [inline images/linux/toolsets/toolset-1604.json:45]: Hello @yuk7! Thank you for the contribution! 
Could you bring these versions (`1.13` and `1.14`) back please? We would like to keep them on the images for now.

Comment (yuk7) [reply images/linux/toolsets/toolset-1604.json]: Thank you for reviewing.
I fixed and pushed them.

**Review by maxim-lobanov — APPROVED** [review]

File: images/linux/toolsets/toolset-1604.json (hunk 8 above)
Comment (yuk7) [reply images/linux/toolsets/toolset-1604.json]: Hello @MaksimZhukov, thank you for reviewing.
I fixed and pushed them.

**Pull request merged**